packages/client/modules/userDashboard/components/OrgBilling/OrgBillingCreditCardInfo.tsx

```diff
@@ -99,6 +100,7 @@ const OrgBillingCreditCardInfo = (props: Props) => {
           <CreditCardModal
             activeUserCount={activeUserCount}
             orgId={orgId}
+            invoiceListRefetch={invoiceListRefetch}
```

Comment:
updating a CC number will have no effect on the invoice list, so this isn't necessary